sounddevice.py

```diff
@@ -232,6 +232,33 @@
 PaError Pa_GetSampleSize( PaSampleFormat format );
 void Pa_Sleep( long msec );
 
+/* pa_mac_core.h */
+
+typedef int32_t SInt32;
+typedef struct
+{
+    unsigned long size;           /**size of whole structure including this header */
+    PaHostApiTypeId hostApiType;  /**host API for which this data is intended */
+    unsigned long version;        /**structure version */
+    unsigned long flags;          /** flags to modify behaviour */
+    SInt32 const * channelMap;    /** Channel map for HAL channel mapping , if not needed, use NULL;*/
+    unsigned long channelMapSize; /** Channel map size for HAL channel mapping , if not needed, use 0;*/
```

Comment:
In the rest of the `cdef`, I've removed those inline comments and moved them to the actual documentation wherever it made sense.